src/wms/src/main/java/org/geoserver/wms/capabilities/CapabilityUtil.java

```diff
@@ -231,4 +268,46 @@ public static String getGroupDefaultStyleName(String groupName) {
     public static String getGroupDefaultStyleName(LayerGroupInfo groupName) {
         return getGroupDefaultStyleName(groupName.prefixedName());
     }
+
+    /** Stores the Min and Max scale denominators for a set of styles. */
+    private static class MinMaxDenominator {
+        private Double minScaleDenominator;
+        private Double maxScaleDenominator;
```

Comment:
Maybe just call them min and max for brevity?

Reply:
Done